fastlane/Fastfile

```diff
@@ -906,6 +928,34 @@ platform :ios do
   end
 end
 
+def extracted_release_notes_file_path(app:)
+  paths = {
+    wordpress: File.join(PROJECT_ROOT_FOLDER, 'WordPress', 'Resources', 'release_notes.txt'),
+    jetpack: File.join(PROJECT_ROOT_FOLDER, 'WordPress', 'Jetpack', 'Resources', 'release_notes.txt')
+  }
+
+  # Good coding would have us handle the key not being found here. But, given
+  # this is a service method for an automation lane, we'd rather have the
+  # process break with a nil value and a human notice it immediately, than an
+  # incorrect file being used and the issue possibly going unnoticed for
+  # longer.
+  paths[app]
```

Comment:
I could also have avoided the `paths` assignment, but the code seemed clearer with it.

```suggestion
  # Good coding would have us handle the key not being found here. But, given
  # this is a service method for an automation lane, we'd rather have the
  # process break with a nil value and a human notice it immediately, than an
  # incorrect file being used and the issue possibly going unnoticed for
  # longer.
  {
    wordpress: File.join(PROJECT_ROOT_FOLDER, 'WordPress', 'Resources', 'release_notes.txt'),
    jetpack: File.join(PROJECT_ROOT_FOLDER, 'WordPress', 'Jetpack', 'Resources', 'release_notes.txt')
  }[app]
```